curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/6401e30d5f45345120a267a3.md

```diff
@@ -0,0 +1,178 @@
+---
+id: 6401e30d5f45345120a267a3
+title: Step 26
+challengeType: 0
+dashedName: step-26
+---
+
+# --description--
+
+Now it's time to extract the part of the `bio` we want. If the `bio` text is more than 50 characters, then we want to extract the first 50 and replace the rest with an ellipsis. The `slice` method is ideal for this use case. Apart from using it to extract the part of an array you want, you can also use it on strings to do the same thing.
+
+You might be thinking you can use `if` statement here, but you can't put an `if` statement inside a template literal. What you have to use instead is a ternary operator, AKA one line `if...else` statement.
+
+Here's the syntax for using the ternary operator:
+
+```js
+condition ? ifTrueExpression : ifFalseExpression;
+```
```

Comment:
While this is a good explanation, I think we could remove this since campers will have plenty of practice using the ternary operator in earlier projects